### PR TITLE
Add additional Solana assets via Picasso IBC

### PR DESCRIPTION
**Pull request by JafarAz**

### CODE REVIEW

File: _non-cosmos/solana/assetlist.json [hunk 1]
```diff
@@ -229,6 +229,301 @@
...
+        },
+        {
+          "denom": "usdt",
+          "exponent": 6
+        }
+      ],
+      "base": "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB",
+      "name": "Tether",
+      "display": "usdt",
+      "symbol": "USDT",
```
Comment (JeremyParish69) [inline _non-cosmos/solana/assetlist.json:513]: please define the trace to ethereum's USDT using additional-mintage.

File: _non-cosmos/solana/assetlist.json [hunk 2]
```diff
@@ -229,6 +229,301 @@
...
+        },
+        {
+          "denom": "jupsol",
+          "exponent": 9
+        }
+      ],
+      "base": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+      "name": "Jupiter Staked SOL",
+      "display": "jupsol",
+      "symbol": "jupSOL",
```
Comment (JeremyParish69) [inline _non-cosmos/solana/assetlist.json:448]: for the Solana LSTs, please define a trace to SOL as type: "liquid-staking"

File: _non-cosmos/solana/assetlist.json [hunk 3]
```diff
@@ -229,6 +229,301 @@
...
+        {
+          "denom": "usdt",
+          "exponent": 6
+        }
+      ],
+      "base": "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB",
+      "name": "Tether",
+      "display": "usdt",
+      "symbol": "USDT",
+      "logo_URIs": {
```
Comment (JeremyParish69) [inline _non-cosmos/solana/assetlist.json:514]: please omit logo_URIs for all new assets--this is a legacy property.

File: _non-cosmos/solana/assetlist.json [hunk 4]
```diff
@@ -229,6 +229,301 @@
...
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/usdt.png"
+      },
+      "coingecko_id": "tether",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/usdt.png"
```
Comment (JeremyParish69) [inline _non-cosmos/solana/assetlist.json:520]: this is a duplication--there are already USDT images in the ethereum/images/ directory. Please remove the new USDT.png here, and point to ethereum's USDT instead using image_sync

Comment (JafarAz) [reply _non-cosmos/solana/assetlist.json]: updated

File: _non-cosmos/solana/assetlist.json (hunk 2 above)
Comment (JafarAz) [reply _non-cosmos/solana/assetlist.json:490]: added

File: _non-cosmos/solana/assetlist.json (hunk 1 above)
Comment (JafarAz) [reply _non-cosmos/solana/assetlist.json:559]: added

File: _non-cosmos/solana/assetlist.json (hunk 3 above)
Comment (JeremyParish69) [reply _non-cosmos/solana/assetlist.json]: yes, please

File: _non-cosmos/solana/assetlist.json [hunk 5]
```diff
@@ -511,15 +581,26 @@
       "name": "Tether",
       "display": "usdt",
       "symbol": "USDT",
-      "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/usdt.png"
-      },
-      "coingecko_id": "tether",
+      "traces": [
+        {
+          "type": "synthetic",
```
Comment (JeremyParish69) [inline _non-cosmos/solana/assetlist.json:562]: sorry, this should be additional-mintage, pointing to Ethereum's USDT.

**Pull request merged**